src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-by-importing-source-code-of-a-project.adoc

```diff
@@ -35,6 +35,39 @@ To create a new workspace to edit an existing codebase, use one of the following
 * xref:#importing-to-a-running-workspace-using-the-git-clone-command_{context}[Import to a running workspace using the `git clone` command]
 * xref:#importing-to-a-running-workspace-with-git-clone-in-a-terminal_{context}[Import to a running workspace using `git clone` in a terminal]
 
+[id="creating-a-custom-workspace-from-the-dashboard"]
+== Select a sample from the Dashboard, then change the devfile to include your project
+
+* In the left navigation panel, go to *Get Started*.
+
+* Click the *Custom Workspace* tab.
```

Comment:
It may be already opened if users already have some workspaces, right? If so, then maybe it makes sense write `Click the *Custom Workspace* tab if it's not already selected.` ?